### ISSUE TITLE
Update reports to use new assertion verdict terminology for priority 3 (May) assertions

### ISSUE THREAD

**Issue by mcking65** [w3c/aria-at-app#1145]
Per agreement in w3c/aria-at#1050, update report tables to express results for "May-Have" behaviors using the word "Supported" instead of "Passed" and "Unsupported" instead of "Failed".

In summary tables, such as [Alert Example | ARIA-AT Reports](https://aria-at.w3.org/report/111731), in the column "May-Have Behaviors", change string from "X of Y Passed" to "X of Y Supported".

In command results tables, such as [JAWS and Chrome for Alert Example | ARIA-AT Reports](https://aria-at.w3.org/report/111731/targets/250#result-NGNhZeyIxMiI6NTcwfQGMyNj), change the string in the "Result" column on rows where priority is "May" to "Supported" instead of "Passed" and "Unsupported instead of "failed".

Note that command results tables appear on:
1. Rreport pages
2. In the candidate review pages
3. In the test runner for tests that are complete.

**Comment by ccanash** [comment]
Noted @mcking65 . We will prioritise it.
